projects/layer-selector/src/lib/expandable-container/expandable-container.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'agrc-expandable-container',
+  template: `
+    <div class="layer-selector" (mouseenter)="expanded = true" (mouseleave)="expanded = false" area-haspopup="true">
```

Comment:
we probably should make this expand on touch events as well for mobile. Not sure if/how that works currently.

Reply:
Confirmed that this does work with touch events as well.